jetty-webapp/src/main/java/org/eclipse/jetty/webapp/ClasspathPattern.java

```diff
@@ -763,7 +762,7 @@ static boolean combine(IncludeExcludeSet<Entry, String> names, String name, Incl
         if (names.hasIncludes() || locations.hasIncludes())
             return byName == Boolean.TRUE || byLocation == Boolean.TRUE;
 
-        // Otherwise there are no includes and it was not excluded, so match
-        return true;
+        // Otherwise there are no includes and it was not excluded, bypass
+        return false;
```

Comment:
This inverts the behavior of IncludeExcludeSet, and would be undesired.

Reply:
May the logic of this part has some problem?